modules/nextflow/src/main/groovy/nextflow/executor/FluxExecutor.groovy

```diff
@@ -77,7 +78,8 @@ class FluxExecutor extends AbstractGridExecutor {
 
         // Time limit in minutes when no units provided
         if( task.config.getTime() ) {
-            result << '--time-limit=' + task.config.getTime().format('mm')
```

Comment:
Why was this removed? This should still work.

```
  -t, --time-limit=MIN|FSD    Time limit in minutes when no units provided, otherwise in Flux standard duration, e.g. 30s, 2d, 1.5h   
```

Reply:
Quoting the original author:

---

`--time-limit` doesn't apply here either. In theory it could be infinite. But our nextflow users have mostly anyway no clue about how long the single flux tasks (nextflow tasks) run. And the hard limit is the Slurm job's wall clock time ... So, I removed that in order to reduce the complexity for the users.

---

I think that this is a misunderstanding by him on how this works, given that the end user should never need to set this manually - it's taken from the pipeline config. So let's leave this in place 👍🏻